microsoft-edge/hosting/webview2/Tooling/WebDriver.md

```diff
@@ -0,0 +1,117 @@
+
+Once you have cloned the repo, build the project in Visual Studio. It should look like the following:
+
+![alt text](..\images\WebDriver\SampleApp.PNG)
+
+## Step 2: Install WebDriver 
+Follow the instructions [here](https://docs.microsoft.com/en-us/microsoft-edge/webdriver-chromium#download-microsoft-edge-driver) to install WebDriver. 
+
+It is important to make sure that the version of WebDriver matches the version of Microsoft Edge that the application uses. For the WebView2API Sample make sure that your version of Edge is greater than or equal to the supported version of our latest SDK release found [here](https://docs.microsoft.com/en-us/microsoft-edge/hosting/webview2/releasenotes). To find out what version of Edge you currently have, load ```edge://settings/help``` in the browser.
+
+## Step 3: Setup Project
+At this point you should have Microsoft Edge installed, have built a WebView2 project, and have installed WebDriver. Now, let’s get started using WebDriver.
+
+1. Start by creating a new **C# .NET Framework** project in **Visual Studio**.
+
+![alt text](..\images\WebDriver\NewProject.PNG)
+
+2. Click **Next** on the bottom right hand corner to continue.
+
+![alt text](..\images\WebDriver\App-Create.PNG)
+
+3. Give your project a **name**, save it to your preferred **location**, and click **Create**.
+
+![alt text](..\images\WebDriver\Start-App.PNG)
+
+4. Now let's add **WebDriver** to the project. You can install WebDriver via the **Selenium.WebDriver NuGet package**. 
+
+To download the **Selenium.WebDriver NuGet package**, in **Visual Studio**, hover over **Project** and select **Manage NuGet Package**. The following screen should appear:
+
+![alt text](..\images\WebDriver\Download-Nuget.PNG)
+
+5. Enter **Selenium.WebDriver** in the search bar, click **Selenium.WebDriver** from the results, and make sure to checkmark the box next to **include pre-release**. On the right hand side window, ensure the **Version** is set to **install 4.0.0-alpha04** and click **Install**. Nuget will download WebDriver to your machine.
+
+To learn more about the **Selenium.WebDriver NuGet package** click [here](https://www.nuget.org/packages/Selenium.WebDriver/4.0.0-alpha04)
+
+![alt text](..\images\WebDriver\Nuget.PNG)
+
+6. Use **OpenQA.Selenium.Edge** by adding the following using statement:```using OpenQA.Selenium.Edge;``` to the top of the file.
+
+```C#
+using OpenQA.Selenium.Edge;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+```
+
+## Step 4: Setup WebDriver
+
+1. First, create the **EdgeOptions** object, by copying the code below. The **EdgeOptions** object takes in two parameters:
+    1. **is_legacy** is set to false, indicating you are using the new Chromium-powered Microsoft Edge Browser.
```

Comment:
I think this is a weird way to show the parameters `EdgeOptions` is taking. I would instead add this:

```markdown
**Parameters**
`is_legacy`: set to `false`, which tells Selenium that you are driving the new Chromium-based Microsoft Edge browser.
`"webview2"`: a string that tell Selenium you are driving **WebView2**
```